gravitee-apim-console-webui/src/management/api/proxy/endpoints/groups/edit/service-discovery/api-proxy-group-service-discovery.component.html

```diff
@@ -0,0 +1,47 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<form *ngIf="!!serviceDiscoveryForm && !!serviceDiscoveryItems" [formGroup]="serviceDiscoveryForm" gioFormFocusInvalid>
+  <mat-card>
+    <div class="card__group-sd__enable">
+      <div class="card__group-sd__enable__label">Enabled service discovery</div>
+      <mat-checkbox aria-label="Enable service discovery" formControlName="enabled">
```

Comment:
I would have expected to have a toggle (with gio wrapper) like on the other screens, what do you think ?

Reply:
same question for the other checkboxes used to activate or not a feature :)

Reply:
Done